data/incipias/incipias first contact.txt

```diff
@@ -269,6 +274,8 @@ mission "Incipias: Help The Stranded 1"
 				`	"I want to help one of your stranded ships."`
 			`	Again a long pause. "Your intention-goals are welcome. Please dock to the capital-station." Along the message comes a small map of the station with the docking ports highlighted.`
 	on complete
+		log `Met with the Incipias on their space station. Gained access to their ships, which are capable of entering the gas giant that the stranded Incipias ship has taken refuge in.`
+		log "Factions" "Incipias" `The Incipias are a species of aliens that live to the northeast of Hai space. They originated on a gas giant, and resemble large, floating octopi. As they have not yet developed the hyperdrive, they are confined to a single system. They are under the protection of the Drak and Quarg, and was warned not to give them any advanced technology or interfere in their development.`
```

Comment:
```suggestion
		log "Factions" "Incipias" `The Incipias are a species of aliens that live to the northeast of Hai space. They originated on a gas giant, and resemble large, floating octopi. As they have not yet developed the hyperdrive, they are confined to a single system. Learned that they are under the protection of the Quarg, and was warned not to give them any advanced technology or interfere in their development.`
```
We don't know it's the Drak (even if we've been in Sayiaban, since we don't ever see it), the only way you would know is if you were hostile with them, which at that point there's nothing further in the area anyways.